services/worker-manager/schemas/v1/worker-full.yml

```diff
@@ -0,0 +1,73 @@
+$schema: "/schemas/common/metaschema.json#"
+title: Worker Full Definition
+description: A complete worker definition.
+type: object
+properties:
+
+  workerPoolId:
```

Comment:
This can also be a `$ref` to `worker-pool-full` I think. Either that or just `$ref` things like `type`/`pattern` and make the description different here.

Reply:
you mean I should reference one field from another schema? I was looking at the `data.js`, it says `workerPoolId: Entity.types.String`, so it doesn't seem to me like you're asking me to reference the whole object

Reply:
like `{$ref: "worker-pool-full.json#/properties/workerPoolId"}`?